docs/RouterConfiguration.md

```diff
@@ -424,7 +430,9 @@ HTTP headers to add to the request. Any header key, value can be inserted.
 // router-config.json
 {
   "configVersion" : "v2.3.0-EN000121",
-  "streetRoutingTimeout" : "5s",
+  "server" : {
+    "apiProcessingTimeout" : "7s"
```

Comment:
apiProcessingTimeout is a server-wide timeout that applies to all API calls while  streetRoutingTimeout is an A* configuration parameter. It may be confusing to put them in the same place.